runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -1196,6 +1196,11 @@ else if (timer == null)
 		{
 			varTimers.put(gameTimer, createGameTimer(gameTimer, duration));
 		}
+		else if (timer.getDuration().compareTo(duration) < 0)
```

Comment:
So, there's one issue with this approach: server ticks do not have a consistent length. Ideally they should fire every 600ms at exactly 600ms intervals but, as we've seen in the past on total level worlds or worlds that are overloaded, tick length varies pretty dramatically. With that in mind, I'd want there to be some leeway so we're not just regularly replacing timers when there's a minuscule time difference between the expected and actual remaining duration.